packages/cf-component-select/example/basic/component.js

```diff
@@ -18,9 +18,9 @@ class Component extends React.Component {
     this.setState({ value });
   };
 
-  handleMultiChange(value) {
+  handleMultiChange(values) {
     this.setState({
-      multiValue: value.length ? value.split(',') : []
+      multiValue: values.map((selected) => selected.value)
```

Comment:
This will only be a comma-separated string if `simpleValue` is true

>The onChange event provides an array of selected options or a comma-separated string of values (eg "1,2,3") if simpleValue is true

From [Multiselect options](https://github.com/JedWatson/react-select#multiselect-options)

But simpleValue is not exposed here so  ¯\_(ツ)_/¯